menu.js

```diff
@@ -1,5 +1,42 @@
 const menu = {
-  // TODO
+  _courses: {
+    appetizers: [],
+    mains: [],
+    desserts: []
+  },
+
+  get courses() {
+    return this._courses;
+  },
+
+  addDishToCourse(courseName, dishName, dishPrice) {
+    const dish = {
+      name: dishName,
+      price: dishPrice
+    };
+    
+    this._courses[courseName].push(dish);
+  },
+
+  getRandomDishFromCourse(courseName) {
+    let randomMealIndex = Math.floor(Math.random() * this._courses[courseName].length);
+    return this._courses[courseName][randomMealIndex];
```

Comment:
You are calling `this._courses[courseName]` 2 times. I think making extra variable for its value is a good idea. Something like `courseDishes` should be fine.
Also, `meal` in this context coming as a set of appetizer, main and dessert dishes. So, I would suggest to rename `randomMealIndex` to `randomDishIndex`.